openshift/factstore.app.yaml

```diff
@@ -92,22 +97,25 @@ objects:
               kind: ImageStreamTag
               name: log-anomaly-detector-factstore:latest
           type: ImageChange
+
   - apiVersion: v1
     kind: Service
     metadata:
-      name: log-anomaly-detector-${APP_NAME}-svc
+      name: log-anomaly-detector-${CUSTOMER_ID}-svc
       labels:
-        app: log-anomaly-detector-factstore-${APP_NAME}
+        app: log-anomaly-detector-factstore-${CUSTOMER_ID}
     spec:
       selector:
-        app: log-anomaly-detector-${APP_NAME}
+        app: log-anomaly-detector-factstore-${CUSTOMER_ID}
       ports:
-        - protocol: TCP
+        - port: 5001
+          name: http
+        - port: 8080
           name: metrics
-          port: 5001
-          targetPort: 5001
+
 parameters:
   - name: SQL_CONNECT
-    value: mysql+pymysql://admin:admin@mysql-56-centos7/factstore
-  - name: APP_NAME
+    value: >
+      mysql+pymysql://admin:password@$(MYSQL_56_CENTOS7_SERVICE_HOST):$(MYSQL_56_CENTOS7_SERVICE_PORT)/factstore
+  - name: CUSTOMER_ID
```

Comment:
@4n4nd Where are you getting `$MYSQL_56_CENTOS7_SERVICE_HOST` or `$MYSQL_56_CENTOS7_SERVICE_PORT` from ?

Reply:
@4n4nd Perhaps it would be easier to revert this back to `mysql+pymysql://admin:admin@mysql-56-centos7/factstore`

Reply:
its the service created when you deploy the sql db

Reply:
You need this to connect to the external sqldb

Reply:
@zmhassan

Reply:
Okay then that would be fine.